doc/release-notes/6.2-release-notes.md

```diff
@@ -0,0 +1,317 @@
+
+### Search by License
+
+A new search facet called "License" has been added and will be displayed as long as there is more than one license in datasets and datafiles in browse/search results. This facet allow you to filter by license such as CC0, etc.
+
+Also, the Search API now handles license filtering using the `fq` parameter, for example : `/api/search?q=*&fq=license%3A%22CC0+1.0%22` for CC0 1.0. 
+
+For more information, see [#10204](https://github.com/IQSS/dataverse/issues/10204).
+
+### Add .QPJ and .QMD Extensions to Shapefile Handling
+
+- Support for **.qpj** and **.qmd** files in shapefile uploads has been introduced, ensuring that these files are properly recognized and handled as part of geospatial datasets in Dataverse.
+
+### Ingested Tabular Data Files Can Be Stored Without the Variable Name Header
+
+Tabular Data Ingest can now save the generated archival files with the list of variable names added as the first tab-delimited line. As the most significant effect of this feature.
+
+Access API will be able to take advantage of Direct Download for tab. files saved with these headers on S3 - since they no longer have to be generated and added to the streamed content on the fly.
+
+This behavior is controlled by the new setting **:StoreIngestedTabularFilesWithVarHeaders**. It is false by default, preserving the legacy behavior. When enabled, Dataverse will be able to handle both the newly ingested files, and any already-existing legacy files stored without these headers transparently to the user. E.g. the access API will continue delivering tab-delimited files **with** this header line, whether it needs to add it dynamically for the legacy files, or reading complete files directly from storage for the ones stored with it.
+
+An API for converting existing legacy tabular files will be added separately. [this line will need to be changed if we have time to add said API before 6.2 is released]. [TODO]
+
+### Uningest/Reingest Options Available in the File Page Edit Menu
+
+New Uningest/Reingest options are available in the File Page Edit menu. Ingest errors can be cleared by users who can published the associated dataset and by superusers, allowing for a successful ingest to be undone or retried (e.g. after a Dataverse version update or if ingest size limits are changed).
+
+The /api/files/<id>/uningest api also now allows users who can publish the dataset to undo an ingest failure.
+
+
+## 🪲 Bug fixes
+
+### Publication Status Facet Restored
+
+In version 6.1, the publication status facet location was unintentionally moved to the bottom. In this version, we have restored the original order.
+
+### Permissions Required To Assign a Role Have Been Fixed
```

Comment:
Not super clear in my opinion, here is a suggestion :
`### Assign a Role With Higher Permissions Than Its Own Role Has Been Fixed`